lib/compiler.js

```diff
@@ -509,3 +545,19 @@ function resolveRelativePaths(relativePaths, appRootDir) {
     }
   });
 }
+
+function normalizeMixinName(str, options) {
+  var normalization = options.normalization;
+  if (normalization === false || normalization === 'none') return str;
+  if (normalization === 'dasherize') {
+    str = String(str).replace(/[\W_]/g, ' ').toLowerCase();
+    str = str.replace(/\s+/g, '-');
+  } else if (typeof normalization === 'function') {
+    str = normalization(str);
+  } else { // classify
+    str = String(str).replace(/[\W_]/g, ' ').toLowerCase();
+    str = str.replace(/(?:^|\s|-)\S/g, function(c) { return c.toUpperCase(); });
+    str = str.replace(/\s+/g, '');
```

Comment:
Also please consider rewriting this using a switch/case + return from each case block, I think the implementation will be simpler/cleaner that way.

```js
if (typeof normalization === 'function') return normalization(str);
switch (normalization) {
  // ...
  case undefined:
  case 'classify':
    // classify & return
  default:
    //report an error
```

How should a `null` value be interpreted? Is it the same as `false`/`'none'`?